README.md

```diff
@@ -14,75 +20,66 @@
     <img src="https://img.shields.io/discord/979116623879368755.svg?label=Immich%20Discord&logo=Discord&style=for-the-badge&logoColor=000000&labelColor=ececec" atl="Immich Discord"/>
   </a>
   <br/>  
-  <br/>  
-  <br/>  
-  <br/>  
-
-  <p align="center">
-    <img src="design/feature-panel.png"  title="Immich Logo">
-  </p>
+  <br/>   
 </p>
 
-# Immich
+## Content
+- [Features](#features)
+- [Screenshots](#screenshots)
+- [Installation](#installation)
+- [Mobile App](#-mobile-app)
+- [Developers](#development)
+- [Support](#support)
+- [Known Issues](#known-issues)
 
-**High performance self-hosted photo and video backup solution.**
-
-![](https://media.giphy.com/media/y8ZeaAigGmNvlSoKhU/giphy.gif)
+# Features 
 
-Loading ~4000 images/videos
+> ⚠️ WARNING: **NOT READY FOR PRODUCTION! DO NOT USE TO STORE YOUR ASSETS**. This project is under heavy development, there will be continuous functions, features and api changes.
 
-## Screenshots
+|  | Mobile | Web |
+| - | - | - | 
+| ☁️ Upload and view videos and photos | Yes | Yes 
+| 🔄 Auto backup when the app is opened | Yes | N/A
+| ☑️ Selective album(s) for backup | Yes | N/A
+| ⬇️ Download photos and videos to local device | Yes | Yes
+| 👪 Multi-user support | Yes | Yes
+| 🖼️ Album | Yes | Yes
+| 🤝 Shared Albums | Yes | Yes
+| 🚀 Quick navigation with draggable scrollbar | Yes | Yes
+| 🗃️ Support RAW (HEIC, HEIF, DNG, Apple ProRaw) | Yes | Yes
+| 🧭 Metadata view (EXIF, map) | Yes | Yes
+| 🔎 Search by metadata, objects and image tags | Yes | No
+| ⚙️ Administrative functions (user management) | N/A | Yes
```

Comment:
I love the emoji here, but it will be hard to find the applicable one for new features

Reply:
So you think it is better to remove them?